www/source/tutorials/getting-started/linux/process-build.html.md.erb

```diff
@@ -84,13 +84,17 @@ To natively run services on your Linux host machine, do the following:
 
    > Note: One of the reasons you can run a service from within the studio is that the `hab` user is created for you when you enter the studio environment.
 
-3. Run `sudo hab install path/to/results/packagefilename.hart`. This will install your package in the same location on the host machine as it does in the studio (i.e. `/hab/pkgs`). Make sure you are in the
+3. Copy the origin public key you created during `hab setup` to `/hab/cache/keys`. Unless your public key has been uploaded to the public depot, you will have to place it in the global key cache before you can install your package for local testing.
+
+       sudo cp ~/.hab/cache/keys/myorigin-20160712195536.pub /hab/cache/keys
```

Comment:
Rather than using cp, I think we should use `hab origin key export myorigin --type public | sudo hab origin key import`.

It abstracts away where the keys live and lets you use `hab` to solve the problem.